### PR TITLE
Feature/pricing plan Pricing Plan Component

### PR DESCRIPTION
**Pull request by nowak-pawel-ds**
Changed Bulma from static import to npm install

### CODE REVIEW

File: core/src/main/java/pl/ds/bulma/components/models/PricingTableComponent.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    private void init() {
+        List<String> classes = new ArrayList<>();
+        if (isComparative) {
+            classes.add("is-comparative");
+        }
+        if (isHorizontal) {
+            classes.add("is-horizontal");
+        }
+
+        pricingClasses = classes.toArray(new String[]{});
```
Comment (arturwujczak) [inline core/src/main/java/pl/ds/bulma/components/models/PricingTableComponent.java:40]: If this is one class then there is no need to use the list here

File: ui.apps/src/main/jcr_root/apps/bulma/components/pricingplan/_ws_dialog/.content.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+          <condition
+              jcr:primaryType="nt:unstructured"
+              sourceName="showButton"
+              values="true"/>
+        </ws:display>
+      </buttonLabel>
+      <variant
+          jcr:primaryType="nt:unstructured"
+          sling:resourceType="wcm/dialogs/components/include"
+          path="/apps/bulma/components/common/variant"/>
```
Comment (arturwujczak) [inline ui.apps/src/main/jcr_root/apps/bulma/components/pricingplan/_ws_dialog/.content.xml]: I am not sure if the current variant list is correct for this component. For example, the value 'is-active' is missing and 'is-primary' is not in the examples for 'pricingtable'. Please check or correct me if I'm wrong.

**Review by arturwujczak — CHANGES_REQUESTED** [review]

File: core/src/main/java/pl/ds/bulma/components/models/PricingTableComponent.java (hunk 1 above)
Comment (nowak-pawel-ds) [reply core/src/main/java/pl/ds/bulma/components/models/PricingTableComponent.java:40]: There are two classes :)

Comment (arturwujczak) [reply core/src/main/java/pl/ds/bulma/components/models/PricingTableComponent.java:40]: Sorry, I assumed that there would always be one of the two.

File: ui.apps/src/main/jcr_root/apps/bulma/components/pricingplan/_ws_dialog/.content.xml (hunk 2 above)
Comment (nowak-pawel-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/pricingplan/_ws_dialog/.content.xml]: Added active type. For the rest I guess it is good to have more options (they work) so author can choose by himself. Also in example there are only 3 options.

Comment (arturwujczak) [reply ui.apps/src/main/jcr_root/apps/bulma/components/pricingplan/_ws_dialog/.content.xml]: Ok, if all the others work, then it can stay.

**Review by arturwujczak — APPROVED** [review]

**Pull request merged**